### PR TITLE
RCB-527 Topics

### PR DESCRIPTION
**Pull request by cp2boston**
- Added endpoint to Api
- Added example
- Added unit test
- Updated example README and source README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 See the wiki for more information.
 
 ## Installation ##
-`composer require "rosette/api: ~1.5"`
+`composer require "rosette/api: ~1.7"`
```
Comment (guruxu) [inline README.md]: we will call it 1.8 when released right?

Comment (cp2boston) [reply README.md]: Yes, although it is just saying that version >= 1.7 and <= 2.0

Comment (guruxu) [reply README.md]: should we stage this to be 1.8 then since it will be required for topics?

Comment (cp2boston) [reply README.md]: OK

**Pull request merged**